docs/ref/models/expressions.txt

```diff
@@ -450,6 +450,116 @@ Conditional expressions allow you to use :keyword:`if` ... :keyword:`elif` ...
 :keyword:`else` logic in queries. Django natively supports SQL ``CASE``
 expressions. For more details see :doc:`conditional-expressions`.
 
+``SubQuery()`` expressions
+--------------------------
+
+.. currentmodule:: django.db.models.expressions
```

Comment:
Do you feel this adds some value? You added aliases in django.db.models so I don't see why these expressions would be treated differently from the rest in this document. I didn't want to change it without asking.